fuzz/Cargo.toml

```diff
@@ -23,10 +24,6 @@ path = "../quinn-proto"
 package = "quinn-proto"
 version = "0.6.1"
 
-# Prevent this from interfering with workspaces
```

Comment:
initializing the fuzz target generates this `Cargo.toml` from a template, and it includes this line. 
Removed now and set as a workspace similar to `bench`, although in so doing I've made tremendous mess in the CI.